usmqe/web/skyring/loginpage/models.py

```diff
@@ -0,0 +1,26 @@
+"""
+Common page model for login page.
+"""
+
+
+from webstr.core import WebstrModel, By, PageElement
+from webstr.common.form import models as form
+
+
+# part of url which is placed after 'usm_web_url'
+# for getting on the described page in the module
+LOCATION = None
```

Comment:
Which code is using (or is expected to use) this global variable? I tried to grep but failed to find it. (the same comment again)